### DISCUSSION TITLE
Cypress doesn't skip 'Uncaught Exception' even though index file in support directory is setup

### DISCUSSION THREAD

**Discussion by RadwaSaleh** [cypress-io/cypress#21390] [CI setup]
What I understood from documentation is that support file `index.js` runs before each spec file

Code in `index.js` file:
```
Cypress.on('uncaught:exception', (err, runnable) => {
  // returning false here prevents Cypress from
  // failing the test
  return false;
});
```
but I still can see the test fail on github actions
<img width="842" alt="Test fail" src="https://user-images.githubusercontent.com/33086179/167398804-0e8d6c8d-2bd9-453e-9c6c-0adbf1c97f65.png">

Test passes locally though
<img width="673" alt="test pass local" src="https://user-images.githubusercontent.com/33086179/167399638-fc72e155-5edf-429c-bcf9-ba9d87c17205.png">

**Comment by muratkeremozcan** [comment]
What happens if you put that code in the relevant spec file vs. the index file?

**Reply by RadwaSaleh** [reply]
> What happens if you put that code in the relevant spec file vs. the index file?

It works correctly when I put it inside the describe block. But I don't want to add this separately per spec file

**Reply by muratkeremozcan** [reply]
Right on. Seems that's the way to use it
